### PR TITLE
Create a new orchestration config to save bundles to ecr-viewer

### PR DESCRIPTION
**Pull request by JNygaard-Skylight**
# PULL REQUEST

## Summary
Created a new orchestration configuration to save a fhir bundle to the database (the same as the current configuration minus conversion) so that if the seed data script finds a directory with just a bundle it can still save it to the database.

## Related Issue
Fixes # n/a

## Acceptance Criteria
1) Add a directory to `containers/ecr-viewer/seed-scripts/baseECR` with a FHIR bundle called `bundle.json`.
2) Run `npm run convert-seed-data `.
3) Check if the eCR from the bundle is in the database.

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/seed-scripts/create-seed-data.py [hunk 1]
```diff
@@ -8,14 +9,29 @@
...
+
+def _process_files(args):
     """
     Convert eICR and RR into FHIR bundles using the FHIR converter.
 
     :return: A list of fhir bundles
     """
     print("Converting files...")
-    subfolders = ["LA"]
+    subfolders = ["ME"]
```
Comment (austin-hall-skylight) [inline containers/ecr-viewer/seed-scripts/create-seed-data.py:34]: This needs to be changed back.

File: containers/ecr-viewer/seed-scripts/create-seed-data.py [hunk 2]
```diff
@@ -8,14 +9,29 @@
...
-def convert_files():
+def _get_args():
+    parser = argparse.ArgumentParser(
+        prog="Create Seed Data",
+        description="Convert eICR and RR files to FHIR bundles and insert them into the database.",
+        epilog="For each directory in baseECR/LA, the script will first look for `bundle.json` files. If a `bundle.json` is found it will be inserted into the database. If there is not a `bundle.json` it will next look for a `CDA_eICR.xml` and `CDA_RR.xml` file. If they are found, it will convert them into a FHIR bundle (saved as `bundle.json`) and insert that into the database using the Orchestration service.",
+    )
+    parser.add_argument(
+        "-f",
+        "--force_convert",
```
Comment (austin-hall-skylight) [inline containers/ecr-viewer/seed-scripts/create-seed-data.py:20]: I like the idea of being able to skip conversion to speed up seeding, but I typically use convert-seed-data to test my FHIR conversion changes so I would lean towards converting by default and having a flag to skip. I'd be interested in hearing the opinions of the other engineers though.

**Review by austin-hall-skylight — CHANGES_REQUESTED** [review]

File: containers/ecr-viewer/seed-scripts/create-seed-data.py (hunk 1 above)
Comment (BobanL) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py:34]: Should we just let it generate everything now?

File: containers/ecr-viewer/seed-scripts/create-seed-data.py (hunk 2 above)
Comment (angelathe) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py:20]: ^ I like this idea!

Comment (BobanL) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py:20]: I would agree with Austin, default should regenerate the bundles and then we can add the flag to skip.

File: containers/ecr-viewer/seed-scripts/create-seed-data.py (hunk 1 above)
Comment (JNygaard-Skylight) [reply containers/ecr-viewer/seed-scripts/create-seed-data.py]: I've reverted this to just LA.

**Review by austin-hall-skylight — APPROVED** [review]

**Pull request merged**